application/templates/mail/request_from_open_to_committed.plush.html

```diff
@@ -1,7 +1,12 @@
-<p>Someone has offered to fulfill a request of yours on <a href="<%= uiURL %>"><%= appName %></a>.</p>
 <h4><a href="<%= postURL %>"><%= postTitle %></a></h4>
 
-<p><strong><%= providerNickname %></strong> has offered to help you with this.
-    (Email: <a href="mailto:<%= providerEmail %>"><%= providerEmail %></a>)</p>
-<br>
-<br>
+<p>
+    Yay! <strong><%= providerNickname %></strong> has offered to help fulfill your request. If you choose to
+    accept their offer, be sure to communicate with them to coordinate the details. For guidance
+    on things to coordinate, see the
+    <a href="">Responsibilities of Requesters</a> section of our Terms of Use.
```

Comment:
```suggestion
    <a href="<%= uiURL %>/#/welcome/terms/responsibilities-of-requesters">Responsibilities of Requesters</a> section of our Terms of Use.
```

Reply:
Is that the right url to the terms? will the page know how to navigate to the right heading with it just in the path like that?

Reply:
Yes, I checked with Matt.